README.md

```diff
@@ -33,8 +33,8 @@ Before configuring Synopsys Security Scan into your azure pipeline, note the fol
 - Sensitive data such as access tokens, user names, passwords and even URLs must be configured using variable groups (Project → Pipelines → Library → New Variable Group)
 
 - `AZURE_TOKEN` is required as input when running Black Duck Fix PR, Black Duck/Coverity PR Comment. There are 2 different types of tokens that can be passed to `AZURE_TOKEN`
-  1. When using `AZURE_TOKEN: $(System.AccessToken)`, you must enable this in the Azure interface. Go to Project → Project Settings → Repository → Security → Build Service and set `Contribute to pull requests` to `Allow`. <br> Confirm `System.AccessToken` has Contribute to PR permissions (Project → Project Settings → Repositories → Security → Build Service User)
-  2. When using `AZURE_TOKEN: $(PAT_TOKEN)`, PAT token should have minimum permissions `Code - Full` and `Pull Request Threads - Read & write`. Refer [Use personal access tokens](https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=Windows) for more details.
+  1. When using `AZURE_TOKEN: $(System.AccessToken)`, confirm it has required permissions to create Pull Requests. Go to Project → Project Settings → Repository → Security → Build Service User and set `Contribute to pull requests`, `Create branch` and `Delete or disable repository` to `Allow`.
```

Comment:
these are not un sync with main branch

Reply:
it is not only for create pull request also for branch creation as part of fix pr

Reply:
i reverted to what was there in main as discused.